.github/workflows/ci.yml

```diff
@@ -10,6 +10,7 @@ on:
 jobs:
   jvm-tests:
     runs-on: ${{ matrix.OS }}
+    if: 1 == 2
```

Comment:
Remove this?

Reply:
This still appear in the first commit. Same for the other `if`s.

Reply:
I have just squashed all commits and remove this change from history